iron-a11y-keys-behavior.html

```diff
@@ -81,14 +90,16 @@
      */
     var SPACE_KEY = /^space(bar)?/;
 
-    function transformKey(key) {
+    function transformKey(key, alphaNumOnly) {
```

Comment:
Could we add a little bit of documentation for this method? I'm particularly interested in documenting the meaning of the `alphaNumOnly` boolean arg.